packages/react-devtools-shared/src/__tests__/setupTests.js

```diff
@@ -80,7 +80,6 @@ beforeEach(() => {
   }
 
   const originalConsoleError = console.error;
-  // $FlowFixMe
```

Comment:
Is there a flowconfig to fail on unused suppressions? Would be nice to continuously remove unused suppressions instead of having to remove them manually.

Reply:
Two things here:
1. `__test__` are just all ignored per flowconfig. Haven't looked into how bad it'd be to change that (we could still do only @flow in some test headers)
2. I think unused suppressions warn, but it's a bit tricky since we have at least a few suppressions that are only suppressing in specific configurations (e.g. only in `yarn flow native`).

Reply:
> we have at least a few suppressions that are only suppressing in specific configurations

Right. Might have similar issues with eslint-disable directives